Source/Public/New-BicepMarkdownDocumentation.ps1

```diff
@@ -0,0 +1,163 @@
+function New-BicepMarkdownDocumentation {
+    [CmdletBinding(DefaultParameterSetName = 'FromFile')]
+    param (
+        [Parameter(ParameterSetName = 'FromFile', Position = 0)]
+        [string]$File,
+
+        [Parameter(ParameterSetName = 'FromFolder', Position = 0)]
+        [string]$Path,
+
+        [Parameter(ParameterSetName = 'FromFolder')]
+        [switch]$Recurse,
+
+        [Parameter(ParameterSetName = 'FromFile')]
+        [Parameter(ParameterSetName = 'FromFolder')]
+        [switch]$Console,
+
+        [Parameter(ParameterSetName = 'FromFile')]
+        [Parameter(ParameterSetName = 'FromFolder')]
+        [switch]$Force
+    )
+
+    switch ($PSCmdLet.ParameterSetName) {
+        'FromFile' { 
+            $FileCollection = @((Get-Item $File)) 
```

Comment:
Instead of forcing Get-Item to return an array, would it maybe make more sense to typecast $FileCollection to an array in both FromFile and FromFolder instead? this may lessen the risk of erroneous types. `[System.IO.FileInfo[]]$FileCollection =`

Reply:
Changed, actually never seemed to have this sort of issue, I think.